content/_getting-started/00-download.md

```diff
@@ -15,3 +15,4 @@ $ git clone https://github.com/SPANDigital/presidium-template.git
 $ git clone https://github.com/SPANDigital/presidium-template-onboarding.git
 $ git clone https://github.com/SPANDigital/presidium-template-design.git
 ```
+**Note**: If you are using the [Docker](/prerequisites/local-environment-setup/#docker) instructions this step is not needed as you clone your repository as a part of the environment setup.
```

Comment:
Do the other links make use of {{ site.baseurl }} ?

Reply:
No, in fact baseurl is not configured in the config.yml (commented out). I originally
put it in and then was confused when it didn't work.